include/crow/websocket.h

```diff
@@ -157,6 +158,36 @@ namespace crow
                     do_write();
                 });
             }
+            
+            /// Send a binary encoded message in blocking mode.
+            void send_binary_blocking(const std::string& msg, boost::system::error_code& ec) override
+            {
+                auto header = build_header(2, msg.size());
+                std::vector<boost::asio::const_buffer> buffer;
+                buffer.emplace_back(boost::asio::buffer(header));
+                buffer.emplace_back(boost::asio::buffer(msg));
+
+                boost::asio::io_service service;
```

Comment:
Would it be possible to use `adaptor_.get_io_service()` instead of creating a new `io_service`?

Reply:
I tried to use it, however, it did not work properly for me. I think there might have been some timing issues where e.g. the deadline_timer is being canceled after write but the async_wait of the timer has not been started yet as the adaptor's IO was too busy?